### ISSUE TITLE
Why 1 - batch_norm_momentum ?

### ISSUE THREAD

**Issue by vfdev-5** [lukemelas/EfficientNet-PyTorch#3]
Thanks for the repo! I was just checking a bit the code and I'm wondering why 

https://github.com/lukemelas/EfficientNet-PyTorch/blob/a558041c0663cc6dc1a8d32bcebf57609ccc0ab3/efficientnet_pytorch/model.py#L31

which seems to be different from the original implementation.

**Comment by lukemelas** [comment]
PyTorch and TensorFlow use different conventions for batch norm momentum. See the note in the official docs: [https://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm2d](https://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm2d)
